testing/cli/test_cli_sct_maths.py

```diff
@@ -1,18 +1,57 @@
 # pytest unit tests for sct_maths
 
+import os
 import pytest
 import traceback as tb
 import logging
 
 import numpy as np
 
-from spinalcordtoolbox.image import Image
+from spinalcordtoolbox.image import Image, compute_dice
 from spinalcordtoolbox.utils.sys import sct_test_path
 from spinalcordtoolbox.scripts import sct_maths
 
 logger = logging.getLogger(__name__)
 
 
+@pytest.mark.parametrize("ofolder", ["tmp_path", None])
+@pytest.mark.parametrize("args_to_test, fname_in, fname_gt", [
+    (["-denoise", "1"],       sct_test_path("t2", "t2.nii.gz"), None),  # TODO: Add gt
+    (["-otsu-median", "5,5"], sct_test_path("t2", "t2.nii.gz"), None),  # TODO: Add gt
+], ids=["-denoise", "-otsu-median"])
+def test_sct_maths_basic(args_to_test, fname_in, fname_gt, ofolder, tmp_path):
```

Comment:
Before I merge, I wanted to just quickly ask you @mguaypaq: How do you feel about the test structure proposed in #4637?

- Before: Individual tests for each argument, even if the overall structure of the tests is more or less the same.
- After: Shared structure between (some) tests, with the arguments being parametrized, leading to (hopefully) quicker tests to fill in missing gaps.

I feel like this works decently for `sct_maths`, and in fact I imagine it could even absorb some of the existing sct_maths tests (e.g. `-percent 95`, `-add 1`, `-symmetrize {0,1,2}`, etc.). I tried this now for kicks and found that `-symmetrize` works:

<details>

```python
def symmetrize_gt(fname_in, dim):
    im_in = Image(fname_in)
    im_out = im_in.copy()
    im_out.data = (im_in.data + np.flip(im_in.data, axis=int(dim))) / 2.0
    fname_out = os.path.join(os.getcwd(), f"symmetrize_{dim}.nii.gz")
    im_out.save(fname_out)
    return fname_out


@pytest.mark.parametrize("ofolder", ["tmp_path", None])
@pytest.mark.parametrize("args_to_test, fname_in, fname_gt", [
    (["-denoise", "1"],       sct_test_path("t2", "t2.nii.gz"), None),  # TODO: Add gt
    (["-otsu-median", "5,5"], sct_test_path("t2", "t2.nii.gz"), None),  # TODO: Add gt
    (["-symmetrize", "0"],    sct_test_path("t2", "t2.nii.gz"), symmetrize_gt(sct_test_path("t2", "t2.nii.gz"), 0)),
    (["-symmetrize", "1"],    sct_test_path("t2", "t2.nii.gz"), symmetrize_gt(sct_test_path("t2", "t2.nii.gz"), 1)),
    (["-symmetrize", "2"],    sct_test_path("t2", "t2.nii.gz"), symmetrize_gt(sct_test_path("t2", "t2.nii.gz"), 2)),
], ids=["-denoise", "-otsu-median", "-symmetrize 0", "-symmetrize 1", "-symmetrize 2"])
```
</details>

Reply:
Sure! I like this structure. And feel free to move existing tests into it if it's easy. (It's fine to have other tests too, if they don't easily fit the mould!)